services/kommander-flux/2.3.0/templates/apps_v1_deployment_helm-controller.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     app.kubernetes.io/component: helm-controller
     app.kubernetes.io/instance: kommander-flux
     app.kubernetes.io/part-of: flux
-    app.kubernetes.io/version: v2.2.3
+    app.kubernetes.io/version: v2.3.0
```

Comment:
https://github.com/fluxcd/flux2/releases shows 2.4.0 is available now

Reply:
same as other comments, trying to keep things in scope given limited time left for 2.13 release.